dolphinscheduler-registry/dolphinscheduler-registry-plugins/dolphinscheduler-registry-jdbc/src/main/java/org/apache/dolphinscheduler/plugin/registry/jdbc/JdbcRegistryConfiguration.java

```diff
@@ -35,6 +35,7 @@
 public class JdbcRegistryConfiguration {
 
     @Bean
+    @ConditionalOnProperty(prefix = "registry", name = "separate-datasource", havingValue = "true")
```

Comment:
Can we use `registry.hikari-config` to do this judge? rather than import `separate-datasource`

Reply:
Yeah,  That's more simple. 
For the user, if they want to use jdbc registry, they need to define just like:
```yaml
registry:
  type: jdbc
```

Reply:
Done, PTAL